ztp/source-crs/ClusterVersion.yaml

```diff
@@ -5,9 +5,10 @@ metadata:
   annotations:
     ran.openshift.io/ztp-deploy-wave: "100"
 spec:
-  # The default channel is stable-<version>
-  channel: $channel
-  # The default upstream path is https://api.openshift.com/api/upgrades_info/v1/graph
-  upstream: $upstream
-  # desiredUpdate:
-  #   version: $version
+  desiredUpdate:
```

Comment:
Should we plan to add more checks here, like looking at the cluster operators' health? Thus ensuring the cluster is in a healthy state after upgrade.

Reply:
Yes we want to add more checks here. This change should address the issue we are having right now.. where the policies are compliant without the upgrade completing..